docs/source/conf.py

```diff
@@ -27,11 +27,12 @@
 sys.path.insert(0, os.path.abspath(PATH_ROOT))
 sys.path.append(os.path.join(PATH_RAW_NB, ".actions"))
 
-_SHOULD_COPY_NOTEBOOKS = True
+_SHOULD_COPY_NOTEBOOKS = bool(int(os.getenv("SHOULD_COPY_PL_NOTEBOOKS", 1)))
 
 try:
-    from helpers import HelperCLI
-except Exception:
+    if _SHOULD_COPY_NOTEBOOKS:
```

Comment:
this if should be outside of the try catch, not the other way around

Reply:
Updated!